kube-monitoring/charts/values.yaml

```diff
@@ -426,7 +427,7 @@ kubeMonitoring:
                 storage: 50Gi
             # -- The storage class to use for the persistent volume
             # @section -- Prometheus options
-            storageClassName: ""
+            storageClassName: gluster
```

Comment:
```suggestion
            storageClassName: cluster
```

Reply:
i was trying out this spec
https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L1021

but neither cluster or gluster are valid options i believe